tests/chainer_tests/optimizers_tests/test_optimizers.py

```diff
@@ -9,8 +9,8 @@
 from chainer import testing
 
 
-@testing.parameterize(*testing.product({
-    'impl': [
+_parameterize = testing.parameterize(*testing.product({
```

Comment:
The variable name should be more specific, like `_parameterize_optimizers`.

Reply:
Changed as suggested.